README.md

```diff
@@ -607,6 +607,32 @@ dsn: clickhouse://dbuser:dbpass@hostname:9000/dbname
 
 See also: https://pkg.go.dev/github.com/ClickHouse/clickhouse-go
 
+**Databricks:**
```

Comment:
```suggestion
**Databricks (Experimental):**
```

As this is a data source I cannot actively maintain and continuous testing via Docker images or similar cannot be performed, please allow me to designate it as ‘Experimental’.